api/processors/actions/group/confirm_ready_to_start.py

```diff
@@ -18,7 +18,7 @@ def _create_log_events(self) -> None:
             teacher_log_event_type=TeacherLogEventType.GROUP_CONFIRMED,
             coordinator_log_event_type=CoordinatorLogEventType.TOOK_NEW_GROUP,
             group_log_event_type=GroupLogEventType.CONFIRMED,
-            from_or_to_group=FromOrToGroup.TO,
+            to_group=self.group,
```

Comment:
```suggestion
```
Let's remove this altogether, as per [this explanation](https://github.com/samanthasgroup/django-webapps/issues/116#issuecomment-1646980857)

Reply:
Fixed